src/utils/arrayUtils.spec.ts

```diff
@@ -64,4 +64,12 @@ describe('arrayUtils', () => {
       expect(result).toEqual(['valid', 1])
     })
   })
+
+  describe('removeDuplicates', () => {
+    it('should remove all duplicates', () => {
```

Comment:
would be nice to check how objects are handled as well

Reply:
also null and undefined

Reply:
Good idea, added